### PR TITLE
Updated prize level column in team view

### PR DESCRIPTION
**Pull request by MiguelTrujillo26**
Updated prize level column in team view.
Fixed some bugs related with the route updatePrizeLevel.

### CODE REVIEW

File: web/src/views/TeamList.js [hunk 1]
```diff
@@ -23,39 +28,68 @@ const TeamList = () => {
...
+    };
+
+    const getTeamList = async () => {
+        if (!actor || !actor.actor) return;
+        const actorId = actor.actor.actor_id;
+        const token = await getAccessTokenSilently();
+        const headers = {
+            'Authorization': `Bearer ${token}`
+        }
+        await axios.get(`http://localhost:4000/v1/team?actorId=${actorId}`,
```
Comment (andymotagl) [inline web/src/views/TeamList.js:67]: Could you please change this param `actorId` to be in the header instead? If you do so, you should update the api documentation too.

File: web/src/views/TeamList.js [hunk 2]
```diff
@@ -23,39 +28,68 @@ const TeamList = () => {
...
-                }
-                if (error.response.status === 401) {
-                    setState({
-                        ...state,
-                        redirect: true
-                    });
-                    return;
-                }
+        )
+        .then(response => setTeamList(response.data.data.length > 0 ? response.data.data : null))
```
Comment (andymotagl) [inline web/src/views/TeamList.js:72]: For list, we return 404 when empty or non exists

**Review by andymotagl — APPROVED** [review]

**Comment by andymotagl** [comment]
Nice job! Amazing implementation. Some small comments

File: web/src/views/TeamList.js (hunk 2 above)
Comment (FernandoArevaloE) [reply web/src/views/TeamList.js:72]: @andymotagl this is used to verify whether table is render or not. Is the same condition we saw yesterday in players, remember that for some reason something like `list.length` is nor working so we need to set the list to null when empty

Comment (andymotagl) [reply web/src/views/TeamList.js:72]: I mean, It is not possible when the response from backend is 2xx and the list is empty, if so, the backend should be adjusted to reflect this behavior.

File: web/src/views/TeamList.js [hunk 3]
```diff
@@ -23,39 +28,68 @@ const TeamList = () => {
...
         redirect: false
     });
+    const [modal, setModal] = useState(false);
+    const [teamToUpdate, setTeamToUpdate] = useState(null);
+    const [selectedLevel, setSelectedLevel] = useState(0);
+    const toggle = (team) => {
+        setModal(!modal);
+        setTeamToUpdate(team);
+        setSelectedLevel(team ? team.prize_level : 0); 
+    };
```
[the remark this answers is not in the document]
Comment (FernandoArevaloE) [reply web/src/views/TeamList.js:38]: maybe is better if modal does not appear if team null for some reason

File: web/src/views/TeamList.js (hunk 2 above)
Comment (FernandoArevaloE) [reply web/src/views/TeamList.js:72]: you're right. 

@MiguelTrujillo26 I'm pretty sure I made that mistake, can you please change `setTeamList(response.data.data.length > 0 ? response.data.data : null)` to `setTeamList(response.data.data)`?

File: web/src/views/TeamList.js (hunk 1 above)
Comment (MiguelTrujillo26) [reply web/src/views/TeamList.js]: Done

File: web/src/views/TeamList.js (hunk 2 above)
Comment (MiguelTrujillo26) [reply web/src/views/TeamList.js]: Done

File: web/src/views/TeamList.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (MiguelTrujillo26) [reply web/src/views/TeamList.js]: Done

**Review by FernandoArevaloE — APPROVED** [review]

**Pull request merged**